melodic/distribution.yaml

```diff
@@ -12858,6 +12858,21 @@ repositories:
       url: https://github.com/SteveMacenski/slam_toolbox.git
       version: melodic-devel
     status: maintained
+  snowbot_operating_system:
+    doc:
+      type: git
+      url: https://github.com/PickNikRobotics/snowbot_operating_system.git
+      version: main
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/PickNikRobotics/snowbot_release.git
+      version: 0.0.1-1
+    source:
+      type: git
+      url: https://github.com/PickNikRobotics/snowbot_operating_system.git
```

Comment:
Kindly asking that you add a top-level LICENSE file to the source repository that matches the declared license in the package.xml. No need to perform another release.

Reply:
License file activated!

Reply:
thanks!